src/contract.rs

```diff
@@ -57,8 +57,14 @@ impl<T: Transport> Instance<T> {
     /// Note that this does not verify that a contract with a matchin `Abi` is
     /// actually deployed at the given address.
     pub fn at(web3: Web3<T>, abi: Abi, address: Address) -> Self {
-        let methods = create_mapping(&abi.functions, |function| function.selector());
-        let events = create_mapping(&abi.events, |event| event.signature());
+        let methods = create_mapping(&abi.functions, |function| Some(function.selector()));
+        let events = create_mapping(&abi.events, |event| {
```

Comment:
Is this related to all events or just for being compatible with anonymous events?

Reply:
I'll split it out.